artifact/parser_test.cpp

```diff
@@ -238,12 +237,12 @@ TEST_F(ParserTestEnv, TestParseEmptyPayloadArtifact) {
 }
 
 TEST_F(ParserTestEnv, TestParseTopLevelSigned) {
-	std::fstream fs {tmpdir->Path() + "/test-artifact-signed.mender"};
+	std::fstream fs {mender::common::path::Join(tmpdir->Path(), "/test-artifact-signed.mender")};
```

Comment:
Beware that absolute path in the second argument won't work with the upcoming https://github.com/mendersoftware/mender/pull/1286, and is bad style nevertheless. Applies to many locations.

Reply:
Btw, you could also shorten to `path::Join` by inserting `namespace path = mender::common::path;` near the top.

Reply:
Thank you.